pkg/asset/templates/content/openshift/rolebinding-cloud-creds-secret-reader.go

```diff
@@ -0,0 +1,64 @@
+package openshift
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/templates/content"
+)
+
+const (
+	rolebindingCloudCredsSecretReaderFileName = "rolebinding-cloud-creds-secret-reader.yaml.template"
+)
+
+var _ asset.WritableAsset = (*RolebindingCloudCredsSecretReader)(nil)
+
+// RolebindingCloudCredsSecretReader is the variable to represent contents of corresponding file
+type RolebindingCloudCredsSecretReader struct {
```

Comment:
nit: throughout this file I think binding should be capitalized `Rolebinding` -> `RoleBinding`

Reply:
ditto